syntax = "proto3";
package sqwak;

service Feature_Extractor {
  rpc GetFeatures (stream AudioFileChunk) returns (FeatureListResponse) {}
}

message AudioFileChunk {
  bytes data = 1;
}

message FeatureListResponse {
  repeated string feature_vector = 4;
}
